backend.py

```diff
@@ -100,6 +111,6 @@ def index():
             total += len(frame_latex[i])
             block.append(frame_latex[i])
             i += 1
-    return json.dumps({'result': block, 'number_of_frames': number_of_frames})
+    return json.dumps({'result': block, 'number_of_frames': number_of_frames, 'height': height, 'width': width, 'total_frames': len(os.listdir(FRAME_DIR))})
```

Comment:
With more complex images, the amount of data is already a lot, so I'm not sure if it is necessary to add these attributes for every request.

Reply:
If you need those values to be sent to the frontend it's probably better to put them in their own request since they should remain constant for all the frames.

Reply:
See I was thinking that but wasn't sure of the proper way to do so, maybe I should check to see if the frame number is 0 and if it is I send back those constants? Just not really sure of the proper way to do so but I'm sure I could figure it out.